Sources/ProjectDescription/Dependencies/ProjectConfiguration.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension ProjectDescription.Project {
+    public struct ProjectConfiguration: Codable, Equatable {
```

Comment:
as it's already namespaced in `ProjectDescription.Project`
```suggestion
    public struct Configuration: Codable, Equatable {
```